index.html

```diff
@@ -8,16 +8,26 @@
         <link rel="stylesheet" href="css/index.css">
     </head>
 
+    <header>
+        <ul class="navbar">
+            <li><a href="#About">About</a></li>
+            <li><a href="#experience">Experience</a></li>
+            <li><a href="#Connect">Contact</a></li>
```

Comment:
```suggestion
            <li><a href="#about">About</a></li>
            <li><a href="#experience">Experience</a></li>
            <li><a href="#connect">Contact</a></li>
```

Probably better to be consistent about capitalization, and make the corresponding changes elsewhere.